### PR TITLE
My First html, CSS and JavaScript Capstone project

### PR DESCRIPTION
**Pull request by gsmalik030**
In this  branch:

I worked on designing the navigation menu for the desktop.
I created team section dynamically using Javascript.
I adjusted the footer section for the about page.
I worked on responsiveness for the website.
I fixed linters errors. 

Video Documentation link: https://www.loom.com/share/23393441feb3404d93d0233cf5da4222

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -479,3 +479,108 @@ hr {
   color: var(--color-accent);
   background-color: var(--color-primary);
 }
+
+@media screen and (min-width: 768px) {
+  .top-bar {
+    display: flex;
+  }
+
```
Comment (HAMMAS-SALEEM) [inline css/styles.css:487]: ### Your Title
title

- [ ] Your title is not descriptive enough. Kindly change your title to something like HTML, CSS, JavaScript Capstone Project to make your title descriptive 😊

Comment (HAMMAS-SALEEM) [inline css/styles.css:487]: ### Your Summary
Capture

### Descriptive Summary
valid

- [ ] Your summary is not descriptive enough. Kindly compare both the images above ⬆️and update your summary 👍

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submissions 🔴

## Hey @gsmalik030 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the requirements but i cannot proceed review on your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @gsmalik030   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to wrap your navigation bars into the `nav` tag. It will increase the user experience and make your website better for screen readers.
   
    - **Your code**
    
     image

- [ ] Kindly make sure to fix the deployed website because it has a lot of errors while using the `Live preview` works fine. I highly recommend fixing the deployment because at this moment it is not working well.

   - **Deployment**
     chrome-capture-2023-0-4 (3)

  - **Live Preview**
  
    chrome-capture-2023-0-4 (1)

- [ ] Kindly make sure to add the correct Live Demo link in your README, at this moment you have the portfolio link there instead of this capstone. Please fix that.

- [ ] Kindly fix the audio problem on your recorded video, at this moment there is no audio in your video. Kindly fix that.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @gsmalik030 , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**